handler/task.go

```diff
@@ -0,0 +1,196 @@
+package handler
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/mr-Evgeny/go_final_project/database"
+	"github.com/mr-Evgeny/go_final_project/model"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+func taskSet(r *http.Request, task *model.ToDo) error {
+	var buf bytes.Buffer
+
+	_, err := buf.ReadFrom(r.Body)
+	if err != nil {
+		return err
```

Comment:
Лучше оборачивать ошибки, чтобы было понятнее откуда она появилась
```suggestion
		return fmt.Errorf("error while reading body, %w", err)
```